common/grpc-helpers/src/lib.rs

```diff
@@ -150,6 +150,6 @@ pub fn convert_grpc_response<T>(
         .and_then(|x| x.map_err(convert_grpc_err))
 }
 
-fn convert_grpc_err(e: ::grpcio::Error) -> Error {
+pub fn convert_grpc_err(e: ::grpcio::Error) -> Error {
```

Comment:
If we make this one public, maybe delete the one at the end of `storage-client/src/lib.rs`.

Reply:
I didn't end up needing this; I'll leave it private to keep the dependencies of storage-client to a minimum.